product/reports/200_Red Hat/001_CPU Right Sizing Recommendations.yaml

```diff
@@ -0,0 +1,95 @@
+---
+title: CPU Right Sizing Recommendations
+rpt_group: Custom
+rpt_type: Custom
+priority:
+db: Vm
+cols:
+- name
+- aggressive_vcpus_recommended_change_pct
+- aggressive_vcpus_recommended_change
+- aggressive_recommended_vcpus
+- moderate_vcpus_recommended_change_pct
+- moderate_vcpus_recommended_change
+- moderate_recommended_vcpus
+- cpu_total_cores
+- num_cpu
+include:
+  ext_management_system:
+    columns:
+    - name
+col_order:
+- name
+- ext_management_system.name
+- aggressive_vcpus_recommended_change_pct
+- aggressive_vcpus_recommended_change
+- aggressive_recommended_vcpus
+- moderate_vcpus_recommended_change_pct
+- moderate_vcpus_recommended_change
+- moderate_recommended_vcpus
+- cpu_total_cores
+- num_cpu
+headers:
+- Name
+- Provider Name
+- CPU - Aggressive Savings (%)
+- CPU - Aggressive Savings
+- 'CPU - Aggressive '
+- CPU - Moderate Savings (%)
+- CPU - Moderate Savings
+- 'CPU - Moderate '
+- CPU Cores
+- CPU Sockets
+conditions: !ruby/object:MiqExpression
+  exp:
+    "=":
+      field: Vm-active
+      value: 'true'
+  context_type:
+order: Descending
+sortby:
+- moderate_vcpus_recommended_change_pct
+group:
+graph:
+dims:
+filename:
+file_mtime:
+categories: []
+timeline:
+template_type: report
+where_clause:
+db_options: {}
+generate_cols:
+generate_rows:
+col_formats:
+-
```

Comment:
Probably these lines are not necessary...

Reply:
Should I delete them?